tests/bearlib/aspects/conftest.py

```diff
@@ -76,3 +78,13 @@ class SubAspect:
         sour = SubAspect_tastes['sour']
 
     return SubAspect
+
+
+@pytest.fixture
+def aspectsTestBear():
+    class testBear(LocalBear, aspects={
```

Comment:
Can+Should also be named `aspectsTestBear`. Hmm... I wonder if you could directly decorate the `class` definition with `@pytest.fixture` in that case...